sevntu-checks/pom.xml

```diff
@@ -113,6 +113,7 @@
                     <regex><pattern>.*.checks.coding.ReturnNullInsteadOfBoolean</pattern><branchRate>80</branchRate><lineRate>88</lineRate></regex>
                     <regex><pattern>.*.checks.coding.SimpleAccessorNameNotationCheck</pattern><branchRate>75</branchRate><lineRate>100</lineRate></regex>
                     <regex><pattern>.*.checks.coding.UnnecessaryParenthesesExtendedCheck</pattern><branchRate>93</branchRate><lineRate>100</lineRate></regex>
+                    <regex><pattern>.*.checks.coding.WhitespaceBeforeArrayInitializerCheck</pattern><branchRate>87</branchRate><lineRate>100</lineRate></regex>
```

Comment:
new Check should appear with 100% coverage , in other words no update is required in pom.xml. That lines are here only for old Checks that existed before we enforced 100%.